contracts/TestDappNFTs.sol

```diff
@@ -1,25 +1,21 @@
 // SPDX-License-Identifier: MIT
 pragma solidity >=0.4.22 <0.9.0;
 
-import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
```

Comment:
Hmm, I tried compiling this in Remix but got an error about `Strings` already being declared. I don't quite understand why, but, it went away with I removed the `@4.8.1` or added it to the other two imports. Perhaps they should all be using the same version of `@openzeppelin/contracts`.